qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -217,14 +217,6 @@ def _build_study_info(user, study_proc=None, proc_samples=None):
     return infolist
 
 
-@execute_as_transaction
```

Comment:
Just want to confirm that this is showing here because `artifact-study-pages` is not up to date with master. I can update this unless you want to do it.